src/util/std_expr.h

```diff
@@ -197,26 +177,24 @@ class decorated_symbol_exprt:public symbol_exprt
   }
 };
 
-/*! \brief Cast a generic exprt to a \ref symbol_exprt
- *
- * This is an unchecked conversion. \a expr must be known to be \ref
- * symbol_exprt.
- *
- * \param expr Source expression
- * \return Object of type \ref symbol_exprt
- *
- * \ingroup gr_std_expr
-*/
+/// \brief Cast a generic exprt to a \ref symbol_exprt
```

Comment:
Is \brief required?

Reply:
It's not required as long as the brief ends with a full stop, which was not the case in the original documents. So instead of wasting time adding full stops in hundreds of comments, I decided to leave `\brief`.

Reply:
If it's not too much of a hassle, could you remove the word `generic` from these comments? It might be confusing, also because we are assuming it's of correct type already.